account_payment_other_company/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,2 @@
+There is no necessary configuration besides having Multi-Compnay enabled.
+*Settings > General Settings*, go to the tab *Multi-Companies* then check *Multi-companies*
```

Comment:
```suggestion
```suggestion
- Multi-company must be enabled: Settings > General Settings, go to the *Multi-Companies* tab, then check *Multi-companies*
- On each company, set the Due To/Due From Account and Journal.
```
```